src/toast/ops/elevation_noise.py

```diff
@@ -71,13 +71,13 @@ class ElevationNoise(Operator):
     noise_a = Float(
         None,
         allow_none=True,
-        help="Parameter 'a' in (a / sin(el) + c)",
+        help="Parameter 'a' in (a / sin(el) + c).  If not set, look for one in the Focalplane.",
```

Comment:
This change is fine, but it does require that the `detector_data` table in the focalplane has names that conform to this.  If not, then the calling code will have to use the current technique of passing in a common value of A / C for all detectors.  Could you also add the A / C column names to the optional columns in the Focalplane docstring here:

https://github.com/hpc4cmb/toast/blob/790f077cdb1d4278902450150e94858c70ad1baf/src/toast/instrument.py#L272

Reply:
Good point! Augmented the `Focalplane` docstring.